specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/privatelinkhub.json

```diff
@@ -0,0 +1,343 @@
+            "schema": {
+              "$ref": "#/definitions/PrivateLinkHub"
+            },
+            "description": ""
+          },
+          "201": {
+            "schema": {
+              "$ref": "#/definitions/PrivateLinkHub"
+            },
+            "description": ""
+          },
+          "default": {
+            "schema": {
+              "$ref": "#/definitions/ErrorContract"
+            },
+            "description": ""
+          }
+        },
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "x-ms-examples": {
+          "Create or update a privateLinkHub": {
+            "$ref": "./examples/CreateOrUpdatePrivateLinkHub.json"
+          }
+        }
+      },
+      "delete": {
+        "operationId": "PrivateLinkHubs_Delete",
+        "description": "Deletes a privateLinkHub",
+        "tags": [
+          "PrivateLinkHubs"
+        ],
+        "deprecated": false,
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/privateLinkHubNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "204": {
+            "description": ""
+          },
+          "default": {
+            "schema": {
+              "$ref": "#/definitions/ErrorContract"
+            },
+            "description": ""
+          }
+        },
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "x-ms-examples": {
+          "Delete a privateLinkHub": {
+            "$ref": "./examples/DeletePrivateLinkHub.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Synapse/privateLinkHubs": {
+      "get": {
+        "operationId": "PrivateLinkHubs_List",
+        "description": "Returns a list of privateLinkHubs in a subscription",
+        "tags": [
+          "PrivateLinkHubs"
+        ],
+        "deprecated": false,
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "schema": {
+              "$ref": "#/definitions/PrivateLinkHubInfoListResult"
+            },
+            "description": ""
```

Comment:
Can you provide more text for the `description`?

Reply:
Description added